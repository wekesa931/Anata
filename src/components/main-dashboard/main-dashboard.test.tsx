
import React from 'react'
import renderer from 'react-test-renderer';
import HNDashboard from './main-dashboard'

describe('<mainDashboard>', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<HNDashboard/>).toJSON()
    expect (tree).toMatchSnapshot()
  })
})