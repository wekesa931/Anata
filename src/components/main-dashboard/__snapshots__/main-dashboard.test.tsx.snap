// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<mainDashboard> should render correctly 1`] = `
<div>
  <div>
    <p
      className="text-heading-2 margin-bottom-16"
      data-testid="hn-text-heading"
    />
    <div>
      <iframe
        frameBorder="0"
        height="960"
        scrolling="yes"
        style={
          Object {
            "background": "transparent",
          }
        }
        title="Airtable view from undefined"
        width="100%"
      />
    </div>
  </div>
</div>
`;
